build.gradle

```diff
@@ -136,9 +136,9 @@ final def quickstepMaxSdk = "35"
 android {
     namespace "com.android.launcher3"
     defaultConfig {
-        // Lawnchair Launcher 14.0 Beta 3
+        // Lawnchair Launcher 15.0 Alpha 1
         // See CONTRIBUTING.md#versioning-scheme
-        versionCode 15_00_00_00
+        versionCode 15_00_01_01
```

Comment:
See my comment above. As this is a technical matter, we can use Lawnchair 15.0 Beta 01 for this.